Webapp/sources/blueprints/reaction_list/routes.py

```diff
@@ -98,3 +101,21 @@ def get_schemata() -> Response:
     )
     schemes = services.reaction.make_scheme_list(reaction_list, size)
     return {"schemes": schemes, "sort_crit": escape(sort_crit)}
+
+
+@reaction_list_bp.route("/get_new_reaction_id", methods=["GET", "POST"])
+@login_required
+def get_new_reaction_id() -> Response:
+    workbook_name = request.json.get("workbook")
+    workgroup_name = request.json.get("workgroup")
+
+    if workgroup_name is None or workbook_name is None:
+        return jsonify('not_found'), status.HTTP_400_BAD_REQUEST
```

Comment:
```suggestion
        return jsonify('Bad Request'), status.HTTP_400_BAD_REQUEST
```
Since the user forgot to put those keys in their request.